### DISCUSSION TITLE
说说证书过期，导致加载不出

### DISCUSSION THREAD

**Discussion by yuazhi** [Qexo/Qexo#577] [General]
![b9cc7eae8909b87366d685ee00e970f8](https://github.com/user-attachments/assets/d99020b1-709d-45e1-998b-2b11c075e7e7)
![ceac73f6d20cdaba3408661a52182f60](https://github.com/user-attachments/assets/013f24a7-15c1-4f71-9d25-d0ed64ceed8e)

**Comment by am-abudu** [comment]
你使用的cdn存在问题，请更换链接
